docs/section-3-creating-parsers.md

```diff
@@ -919,6 +922,23 @@ It is good practice to explicitly erase your scanner state variables at the star
 
 #### Scan
 
+This function is responsible for recognizing external tokens. It should return `true` if a token was recognized, and `false` otherwise. It is called with a "lexer" struct.
```

Comment:
Question for the maintainers:

Should "lexer" be inside quotes as it currently stands, or inside back ticks?

Reply:
Thanks for the quick response.

I kept it in quotes here as it seems like a categorization rather than a name. Later I use backticks to make clear it matches the `lexer` argument to `tree_sitter_my_language_external_scanner_scan`.